src/etc/inc/captiveportal.inc

```diff
@@ -1200,14 +1205,19 @@ function captiveportal_disconnect($dbent, $term_cause = 1, $stop_time = null) {
 		}
 	}
 
-	// XMLRPC Call over to the master Voucher node
-	if (xmlrpc_sync_voucher_details($syncip, $syncport,
-	    $vouchersyncusername, $syncpass)) {
-		$remote_status = xmlrpc_sync_voucher_disconnect($dbent, $syncip,
-		    $syncport, $syncpass, $vouchersyncusername, $term_cause,
-		    $stop_time);
-	}
+	// XMLRPC Call over to the backup node if necessary
+	if (xmlrpc_sync_get_details($syncip, $syncport,
+	    $syncuser, $syncpass, $carp_loop)) {
+		$rpc_client = new pfsense_xmlrpc_client();
+		$rpc_client->setConnectionData($syncip, $syncport, $syncuser, $syncpass);
+		$rpc_client->set_noticefile("CaptivePortalUserSync");
+		$arguments = array('sessionid' => $dbent[5],
+		'term_cause' => $term_cause,
+		'stop_time' => $stop_time);
 
+		$rpc_client->xmlrpc_method('captive_portal_sync', array('op' => 'disconnect_user', 'zone' => $cpzone, 'session' => base64_encode(serialize($arguments))));
```

Comment:
This line is very long, either move the `array(...` part to a new line, or define the array before using it. I know we prefer to avoid single use variables but in this case it would make it much easier to read/follow.